x/auth/module.go

```diff
@@ -203,7 +203,7 @@ func init() {
 	)
 }
 
-type AuthInputs struct {
+type Inputs struct {
```

Comment:
I understand why you do that, however, imho, I find it more readable the way we had.
This is only used by depinject so we actually never import `auth.AuthInputs`, so the duplication of name isn't relevant.

Could you please revert them all?

Reply:
oh sure!  Actually, this wasn't supposed to change -- I reverted some, I'll check throught the rest...

Reply:
Thanks!